GeneralStateTests/stCallCreateCallCodeTest/createNameRegistratorPerTxsNotEnoughGas.json

```diff
@@ -2,10 +2,11 @@
     "createNameRegistratorPerTxsNotEnoughGas" : {
         "_info" : {
             "comment" : "",
-            "filledwith" : "cpp-1.3.0+commit.619a6438.Linux.g++",
+            "filledwith" : "cpp-1.3.0+commit.84983edf.Linux.g++",
+            "retesteth" : "cpp-0.0.1+commit.b346d8e6.Linux.g++",
```

Comment:
What?

Reply:
retesteth commit hash

Reply:
I see. But can you rename it to something more generic. E.g. "client_version" and "tool_version". And why retesteth is called "cpp"?

Reply:
`tool_version` does not look more generic then retesteth
`cpp` states for cpp implementation

Reply:
@winsvega can you make it, something like

```
"filling-rpc-host-version" : "cpp-1.3.0+commit.84983edf.Linux.g++",
"filling-runner" : "retesteth-0.0.1+commit.abdcda.Linux.g++"
```

Reply:
@winsvega let's prepare for alternative implementations of `retesteth` and let's assume not all of them are called `retesteth`.

Reply:
`filling-rpc-client`
`filling-tool-version`

Reply:
@winsvega For consistency, you either add `-version` suffix or not.

Reply:
@winsvega  `cpp-ethereum` is the RPC server, not the client.  `retesteth` sends requests (client) and `cpp-ethereum` answers (server).